### PR TITLE
*: v0.20.19 release notes

### PR DESCRIPTION
**Pull request by tilacog**

### CODE REVIEW

File: packages/indexer-agent/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Contract interface changes involving Exponential Rebates
+- Included the protocolNetwork filed when checking for duplicate action targets
```
Comment (pcarranzav) [inline packages/indexer-agent/CHANGELOG.md:10]: nit: filed -> field?

File: packages/indexer-agent/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Added
+- Enforced active connections with Graph-Node and the database during startup
```
Comment (pcarranzav) [inline packages/indexer-agent/CHANGELOG.md:13]: Is this the connection pool fix? If so might be mentioning explicitly that we now allow configuring the db connection pool?

**Review by pcarranzav — APPROVED** [review]
Some details but otherwise LGTM

**Review by pcarranzav — APPROVED** [review]

**Comment by tilacog** [comment]
Rebased on top of latest `main` so CI passes

**Pull request merged**